From penberg@cs.helsinki.fi  Wed Nov 12 13:43:10 2008
From: Pekka J Enberg <penberg@cs.helsinki.fi>
Date: Wed, 29 Oct 2008 20:12:08 +0200 (EET)
Subject: Staging: w35und: remove unused link status code
To: Greg Kroah-Hartman <greg@kroah.com>
Cc: Pavel Machek <pavel@suse.cz>
Message-ID: <Pine.LNX.4.64.0810292011500.16596@melkki.cs.Helsinki.FI>


From: Pekka Enberg <penberg@cs.helsinki.fi>

The WBLINUX_ConnectStatus() and related code is not used anywhere so remove
them from the driver.

Acked-by: Pavel Machek <pavel@suse.cz>
Signed-off-by: Pekka Enberg <penberg@cs.helsinki.fi>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/winbond/adapter.h      |    6 ------
 drivers/staging/winbond/linux/common.h |    4 ----
 drivers/staging/winbond/mds_s.h        |    1 -
 drivers/staging/winbond/wblinux.c      |    6 ------
 drivers/staging/winbond/wblinux_f.h    |    1 -
 5 files changed, 18 deletions(-)

--- a/drivers/staging/winbond/adapter.h
+++ b/drivers/staging/winbond/adapter.h
@@ -1,9 +1,5 @@
-#define OS_LINK_STATUS			(adapter->LinkStatus == OS_CONNECTED)
 #define OS_SET_SHUTDOWN( _A )		_A->shutdown=1
 #define OS_SET_RESUME( _A )		_A->shutdown=0
-#define OS_CONNECT_STATUS_INDICATE( _A, _F )		WBLINUX_ConnectStatus( _A, _F )
-#define OS_DISCONNECTED	0
-#define OS_CONNECTED	1
 #define OS_STOP( _A )	WBLINUX_stop( _A )
 
 #define OS_CURRENT_RX_BYTE( _A )		_A->RxByteCount
@@ -37,8 +33,6 @@ struct wb35_adapter {
 
 	atomic_t ThreadCount;
 
-	u32 LinkStatus;		// OS_DISCONNECTED or OS_CONNECTED
-
 	u32 RxByteCount;
 	u32 TxByteCount;
 
--- a/drivers/staging/winbond/linux/common.h
+++ b/drivers/staging/winbond/linux/common.h
@@ -44,10 +44,6 @@
 #define WBDEBUG( _M )	0
 #endif
 
-#define OS_DISCONNECTED	0
-#define OS_CONNECTED	1
-
-
 #define OS_EVENT_INDICATE( _A, _B, _F )
 #define OS_PMKID_STATUS_EVENT( _A )
 
--- a/drivers/staging/winbond/mds_s.h
+++ b/drivers/staging/winbond/mds_s.h
@@ -25,7 +25,6 @@
 #define CURRENT_CONTROL_PORT_BLOCK		( psSME->wpa_ok!=1 || (adapter->Mds.boCounterMeasureBlock==1 && (CURRENT_ENCRYPT_STATUS==ENCRYPT_TKIP)) )
 #define CURRENT_FRAGMENT_THRESHOLD		(adapter->Mds.TxFragmentThreshold & ~0x1)
 #define CURRENT_PREAMBLE_MODE			psLOCAL->boShortPreamble?WLAN_PREAMBLE_TYPE_SHORT:WLAN_PREAMBLE_TYPE_LONG
-#define CURRENT_LINK_ON					OS_LINK_STATUS
 #define CURRENT_TX_RATE					adapter->sLocalPara.CurrentTxRate
 #define CURRENT_FALL_BACK_TX_RATE		adapter->sLocalPara.CurrentTxFallbackRate
 #define CURRENT_TX_RATE_FOR_MNG			adapter->sLocalPara.CurrentTxRateForMng
--- a/drivers/staging/winbond/wblinux.c
+++ b/drivers/staging/winbond/wblinux.c
@@ -224,9 +224,3 @@ error:
 
 	return false;
 }
-
-void WBLINUX_ConnectStatus(struct wb35_adapter * adapter, u32 flag)
-{
-	adapter->LinkStatus = flag; // OS_DISCONNECTED	or  OS_CONNECTED
-}
-
--- a/drivers/staging/winbond/wblinux_f.h
+++ b/drivers/staging/winbond/wblinux_f.h
@@ -14,6 +14,5 @@ void wb35_set_multicast( struct net_devi
 struct net_device_stats * wb35_netdev_stats( struct net_device *netdev );
 void WBLINUX_stop(  struct wb35_adapter *adapter );
 void WbWlanHalt(  struct wb35_adapter *adapter );
-void WBLINUX_ConnectStatus(  struct wb35_adapter *adapter,  u32 flag );
 unsigned char WbWLanInitialize(struct wb35_adapter *adapter);
 
